web/client/codechecker_client/cmd_line_client.py

```diff
@@ -403,7 +403,13 @@ def check_run_names(client, check_names):
     run_filter.names = check_names
     run_filter.exactMatch = check_names is not None
 
-    run_info = {run.name: run for run in get_run_data(client, run_filter)}
+    run_info = {}
```

Comment:
I don't really like this change. We are handling exceptions which come from the server side in the [thrift_call.py](https://github.com/Ericsson/codechecker/blob/master/web/client/codechecker_client/thrift_call.py) file globally.

So in this file for example we will catch the `RequestFailed` exceptions, we print the message and raise the exception again, and we catch these exception again where we called the API functions.
Maybe it would be a better approach that after we print the error message we exit with an error code.
If an error happens, for example I am not able to fetch the runs from the server, I want to know it and if I am running these command for example in a Jenins job than the job will fail. And this is the expected behavior instead of working with false data.

In this patch sometimes we will exit with an error code and sometimes we will not (for example in this place if getting the run data is failed we will print an error message but the will not stop the program). I think this is not a good approach.

If the error messages are not expressive, we should fix these messages on the server side where we throw these exceptions and in the thrift_call.py file we should print these messages nicely and exit with an error code.

Reply:
I've updated the PR with the changes we discussed.